internal/file/file.go

```diff
@@ -42,9 +42,11 @@ func Open(path string, flg int, perm os.FileMode) (*os.File, error) {
 			}
 		}
 
-		file, err = os.Create(path)
-		if err != nil {
-			return nil, err
+		if flg&(os.O_CREATE|os.O_APPEND) > 0 {
+			file, err = os.Create(path)
+			if err != nil {
+				return nil, err
+			}
```

Comment:
Did you add test cases for this change?

Reply:
sorry I haven't added test case for this change.
do you think it is better to include it in this PR?

Reply:
i thinks so.

Reply:
do you think this test case is enough?
https://github.com/vdaas/vald/blob/master/internal/file/file_test.go#L90

Reply:
I think it's not enough, we should test a case that no file with an error will be returned when there's no file in the passed `path` and the passed `flg` has neither `O_CREATE` nor `O_APPEND`.

Reply:
yeah you are right! I will fix it today